### ISSUE TITLE
Problem to import deeplabcut: Intel MKL FATAL ERROR: Cannot load mkl_intel_thread.dll.

### ISSUE THREAD

**Issue by rustlab1** [DeepLabCut/DeepLabCut#500]
Hi together, 

I am using Windows 10, GPU (Geforce rtx 2080 ti) with Anaconda Env (with Python 3.7) & DeepLabCut 2.1. 

**Describe the problem**

I am following the Easy Install Windows Instruction from the DeepLabCut website (http://www.mousemotorlab.org/deeplabcut). Everything works for me until I try to import deeplabcut. I get two errors (see screenshots). 

1) A pop-up window that says:  "The ordinal 242 could not be located in the dynamic link library C:\users........\dlc-windowsGPU\Library\bin\mkl-intel_thread.dll. 

2)  In the Terminal window:
INTEL MKL ERROR: The operating system cannot run %1. mkl_intel_thread.dll.
Intel MKL FATAL ERROR: Cannot load mkl_intel_thread.dll.

If someone has an idea, this would be a great help! 

With kind regards,
Ruslan 

![Capture-22](https://user-images.githubusercontent.com/58003304/69262189-51f8a780-0bc3-11ea-89bf-19aadbacf756.PNG)

**Comment by wi11iamk** [comment]
I had this error the other day as well. I've read that the error may be due to other MKL-related DLLs being missing. This thread explains it more clearly: https://stackoverflow.com/questions/57567892/intel-mkl-fatal-error-cannot-load-mkl-intel-thread-dll. If this is the case, the problem may have to do with Windows 10 updates or Python 3.7 implementations (which I have also read may cause some conflicts with MKL). This is by no means a definitive answer. I've just read some on the same problem since last week. Good luck!

**Comment by MMathisLab** [comment]
inside the env, can you please run: ``pip install numpy==1.16.4``

**Comment by rustlab1** [comment]
Many thanks for the replies! Unfortunately, pip install numpy==1.16.4 inside the env did not solve the problem, I get still the same error.

**Comment by MMathisLab** [comment]
Sure, but the warnings you get from importing dlc should be gone. Was @liamwilt  link helpful?

**Comment by rustlab1** [comment]
Many thanks for the reply. The reply did unfortunately not help. I now switched back to the version 2.0 of deeplabcut. And I am a bit further with the installation. The error I am now facing is:

1) After extracting the frames I get the following error: 
ModuleNotFoundError                       Traceback (most recent call last)
ModuleNotFoundError: No module named 'numpy.core._multiarray_umath'

I tried to upgrade numpy (as it is suggested by some others) to different versions. However if I understand correctly I need the numpy==1.14.5 for DLC 2.0 . Anyone else facing this problem? 

Cheers,
Ruslan
